apps/web/.env.example

```
@@ -0,0 +1,5 @@
+NEXT_PUBLIC_BASE_URL=http://localhost:3000
+GITHUB_TOKEN=ghp_****************************9KRLs
```

dotenv-linter (4.0.0)

[warning] [UnorderedKey] The GITHUB_TOKEN key should go before the NEXT_PUBLIC_BASE_URL key

(UnorderedKey)